cogs/plant_shop_commands.py

```diff
@@ -42,6 +46,11 @@ async def get_available_plants(self, user_id:int) -> dict:
                 available_plants = {}
                 for level, plants in possible_available_plants.items():
                     available_plants[level] = random.choice(plants)
+                    
+                    # Make sure users don't get the same plant twice
+                    while available_plants[level] == current_plants[level]:
```

Comment:
New users won't have a `current_plants` defined.

Reply:
Added a check for plant_rows